Source/GNUmakefile

```diff
@@ -684,19 +684,6 @@ GSXibElement.h \
 GSXibLoading.h \
 GSXibKeyedUnarchiver.h \
 GSHelpAttachment.h \
```

Comment:
You should remove the trailing backslash as well.

Reply:
it is convenient to add later, but yes, cleaner. Removed.